PetScanList/wikitable.py

```
@@ -12,61 +11,51 @@
-        for x in table_head:
-            if x == "#":
-                formated_x = str(n)
-            elif x == "touched":
-                formated_x = time.strftime("%Y-%m-%d %H:%M:%S", time.strptime(data.get(x), "%Y%m%d%H%M%S"))
-            else:
-                formated_x = head_formats.get(x, "{}").format(data.get(x, ""))
-            row.append(formated_x)
-        # ---
-        row = "! " + "\n| ".join(row)
-        # ---
+            table_head = table_head2 + [x for x in data.keys() if x not in table_head2]
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)